File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Java_Script/questao06.js
numero = parseInt(prompt("Digite um número entre 1 e 10 para ver sua tabuada: "))

if (numero > 0 && numero < 11) {
    console.log("Adição")
    for ( step = 1; step <11; step++) {
        var adi = numero + step
        console.log(`${numero} + ${step}: ${adi}`)
    }

    console.log("Subtração")
    for ( step = 1; step <11; step++) {
        var sub = numero - step
        console.log(`${numero} - ${step}: ${sub}`)
    }

    console.log("\nMultiplicação")
    for ( step = 1; step <11; step++) {
        var mult = numero * step
        console.log(`${numero} x ${step}: ${mult}`)
    }

    console.log("\nDivisão")
    for ( step = 1; step <11; step++) {
        var div = numero / step
        console.log(`${numero} / ${step}: ${div.toFixed(2)}`)
    }
}

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao06.kt
fun main(){
    println("Digite um número inteiro entre 1 e 10)
    var numero = readLine()!!.toInt()

    if (numero > 0 && numero < 11) {
        println("Adição")
        for (step in 1..10) {
            var adi = numero + step
            println("$numero + $step: $adi")
        }

        println("Subtração")
        for (step in 1..10) {
            var sub = numero - step
            println("$numero - $step: $sub")
        }

        println("\nMultiplicação")
        for (step in 1..10) {
            var mult = numero * step
            println("$numero * $step: $mult")
        }

        println("\nDivisão")
        for (step in 1..10) {
            var div = numero / step
            println("$numero / $step: $div")
        }
    }
}